Sources/ArcGISToolkit/Components/Offline/OfflineMapAreasView.swift

```diff
@@ -14,22 +14,28 @@
 
 import SwiftUI
 import ArcGIS
+import Network
 
 /// The `OfflineMapAreasView` component displays a list of downloadable preplanned map areas from a given web map.
 @MainActor
 @preconcurrency
 public struct OfflineMapAreasView: View {
     /// The view model for the map.
     @StateObject private var mapViewModel: MapViewModel
-    
+    /// The network monitor.
+    @StateObject private var networkMonitor = NetworkMonitor()
     /// The action to dismiss the view.
     @Environment(\.dismiss) private var dismiss: DismissAction
-    
     /// The web map to be taken offline.
     private let onlineMap: Map
-    
     /// The currently selected map.
     @Binding private var selectedMap: Map?
+    /// A Boolean value indicating whether the device connection has been determined.
+    @State private var connectionIsDetermined = false
+    /// A Boolean value indicating whether the device has an internet connection.
```

Comment:
Does this doc need to be updated? It still sounds like it is just the inverse of `offlineBannerIsPresented`.